Sources/Apollo/RequestBodyCreator.swift

```diff
@@ -25,10 +25,10 @@ extension RequestBodyCreator {
   ///   - sendQueryDocument: Whether or not to send the full query document. Defaults to true.
   ///   - autoPersistQuery: Whether to use auto-persisted query information. Defaults to false.
   /// - Returns: The created `GraphQLMap`
-  public func requestBody<Operation: GraphQLOperation>(for operation: Operation,
-                                                       sendOperationIdentifiers: Bool = false,
-                                                       sendQueryDocument: Bool = true,
-                                                       autoPersistQuery: Bool = false) -> GraphQLMap {
+  public func defaultRequestBody<Operation: GraphQLOperation>(for operation: Operation,
```

Comment:
why you rename this method?

Reply:
Hi! Please read the PR description for a full description of all these changes and why they've been made.